app/main/views/index.py

```diff
@@ -72,11 +72,28 @@ def index():
 
 @main.route('/robots.txt')
 def robots():
+    robot_rules = [
+        'User-agent: *',
+        'Disallow: /sign-in',
+        'Disallow: /contact',
+        'Disallow: /register'
+    ]
     return (
-        'User-agent: *\n'
-        'Disallow: /sign-in\n'
-        'Disallow: /contact\n'
-        'Disallow: /register\n'
+        '\n'.join(robot_rules)
+    ), 200, {'Content-Type': 'text/plain'}
+
+
+@main.route('/.well-known/security.txt')
+def security_txt():
+    security_info = [
+        'Contact: mailto:' + current_app.config["SECURITY_EMAIL"],
```

Comment:
```suggestion
        f'Contact: mailto:{current_app.config["SECURITY_EMAIL"]}',
```

If you like f-strings, you can use this syntax. Really not important, just a suggestion

Reply:
Read a little more into using f-strings and it seems the consensus is that it tends to be more read-able. As someone who likes to understand what they read, I'll be switching to using f-strings here!